index.html

```diff
@@ -113,14 +87,14 @@
         }
 
         function sendEmail() {
-            getAccessToken(applicationConfig.graphScopes, function (token, error) {
-                if (token) {
+            clientApplication.acquireTokenSilent(applicationConfig.graphScopes)
```

Comment:
What would happen when the token has expired? or if there is a need for interaction (Conditional access for instance). I don't have the impression that we can do without also, in some cases calling acquireToken?

We could call the previous getToken function acquireTokenWithInteractionsWhenNeeded or something like that ? but it seems needed?